src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -3486,13 +3494,6 @@ class ShowInterfaceTransceiverDetailSchema(MetaParser):
                 Optional('serial_number'): str,
                 Optional('transceiver'): str,
                 Optional('type'): str,
-                Any(): {
-                    Optional('Value'): float,
-                    Optional('HighAlarmThreshold'): float,
-                    Optional('HighWarnThreshold'): float,
-                    Optional('LowWarnThreshold'): float,
-                    Optional('LowAlarmThreshold'): float,
```

Comment:
These schema values were deleted, but they're still being referenced in ShowInterfaceTransceiverDetail, starting line 3563. Was this intentional?

Reply:
Code is there now. Resolved